bin/src/MixinGenerator.php

```diff
@@ -0,0 +1,435 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webmozart\Assert\Bin;
+
+use InvalidArgumentException;
+use ReflectionClass;
+use ReflectionException;
+use ReflectionMethod;
+use RuntimeException;
+use Webmozart\Assert\Assert;
+
+final class MixinGenerator
+{
+    /**
+     * @psalm-var list<string>
+     *
+     * @var string[]
+     */
+    private $unsupportedMethods = [
+        'nullOrNotInstanceOf',  // psalm bug, should be reported
```

Comment:
Have tickets been opened for the methods listed here?

Reply:
Oh, I totally forgot about it. Will do very soon.

Reply:
Done.